tracks/python/exercises/reverse-string/mentoring.md

```diff
@@ -0,0 +1,74 @@
+### Problem and challenges
+
+The problem requires the student to reverse a given string.
+
+### Reasonable solutions
+
+[Extended Slice Notation][1] is the fastest way to reverse a string in Python
+
+```python
+def reverse(text):
+    return text[::-1]
+```
+
+Though it can be tricky for beginners to comprehend, also some developers
```

Comment:
Awkward grammar here, perhaps "Though it can be tricky for beginners to comprehend and some developers
consider it less readable, so"?